### ISSUE TITLE
Hotspot partial display on navigation

### ISSUE THREAD

**Issue by archit1508** [naver/egjs-view360#321]
## Description

While navigating on the site, the hotspots which are placed at a distance show up partially (image attached). The partial hotspot shown is actually at the other end of the page. This might be an issue with the location setting of the hotspots. If anyone has managed to fix the issue please suggest the solution. Thanks
<img width="1277" alt="Screenshot 2020-08-18 at 2 42 58 AM" src="https://user-images.githubusercontent.com/22590085/90444979-81a9f100-e0fc-11ea-878e-1928acd29006.png">

**Comment by WoodNeck** [comment]
Hello, @archit1508 !
Can you show me a reproducible code for that issue? You can use [Codepen](https://codepen.io/).

**Comment by archit1508** [comment]
Hello @WoodNeck! 
Here is the link for the modified hotspot example code pen. Even here the museum images stay stuck like the above picture.
https://codepen.io/archit1508/pen/ZEWOoQe?editors=1010

**Comment by WoodNeck** [comment]
As your image has a size of 250px and the script set image's offset to -200px  when it's out of sight, the image is displayed at upper corner.
So, you can set offset to a higher value to make it out of the canvas, or you can explicitly set it to `visibility: hidden` or `display: none` to make sure it's invisible.

tl;dr;
```diff
    // Line 45
    if (Math.abs(deltaYaw) > 90) {
-        hotspot.style.transform = "translate(-200px, 0px)";
+        hotspot.style.transform = "translate(-400px, 0px)";
        return;
    }
```

Or

```diff
    // Line 45
    if (Math.abs(deltaYaw) > 90) {
-       hotspot.style.transform = "translate(-200px, 0px)";
+       hotspot.style.display = "none";
        return;
+   } else {
+       hotspot.style.display = "block";
+   }
```

**Comment by archit1508** [comment]
@WoodNeck It worked. Thank you so much! :D
